### PR TITLE
Keep consistentcy between charts/catalog/README and README

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

1. According to  #1188,  it doesn't work fine by https. 
   modify URL from https to http
2. Keep consistentcy between charts/catalog/README and README

### CODE REVIEW

**Comment by yuexiao-wang** [comment]
@duglin PTAL

**Comment by duglin** [comment]
LGTM

File: charts/catalog/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Service Catalog is a Kubernetes Incubator project that provides a
 Kubernetes-native workflow for integrating with 
-[Open Service Brokers](https://www.openservicebrokerapi.org/) 
+[Open Service Broker API](http://www.openservicebrokerapi.org/)
```
Comment (duglin) [inline charts/catalog/README.md]: Actually, I think the old text was more correct. Can you change it back to "Brokers"?

Comment (yuexiao-wang) [reply charts/catalog/README.md]: OK, maybe I understand it incorrectly.

**Comment by yuexiao-wang** [comment]
@duglin  It is done

**Comment by duglin** [comment]
I just found out that the URLs with "www" in them works ok with https - so I think the only change from this PR that might still be needed is the last one.  Which is why the URLs you modified never broke the build.

Note: https://github.com/kubernetes-incubator/service-catalog/pull/1190

**Comment by yuexiao-wang** [comment]
@duglin Got it and it is coming back.
PTAL

**Comment by duglin** [comment]
thanks
LGTM

**Pull request merged**